tests/lib/prepare-uc20.sh

```diff
@@ -0,0 +1,224 @@
+#!/bin/bash
+
+set -e
+set -x 
+
+# these should already be installed in GCE and LXD images with the google/lxd-nested 
+# backend, but in qemu local images from qemu-nested, we might not have them
+if [ "${SPREAD_BACKEND}" = "qemu-nested" ]; then
```

Comment:
(nitpick) calling apt install on things already installed will not hurt so we don't strictly need this if but it's fine, does not hurt the other way either.